src/cursor.ts

```diff
@@ -216,11 +222,16 @@ export async function executeWithCursorPagination<
 
   const reversed = !!opts.before && !opts.after;
 
-  for (const { expression, direction } of fields) {
-    qb = qb.orderBy(
-      expression,
-      reversed ? (direction === "asc" ? "desc" : "asc") : direction,
-    );
+  for (const { expression, direction, nullOrder } of fields) {
+    let dir = reversed ? (direction === "asc" ? "desc" : "asc") : direction;
+
+    if (nullOrder === "NULLS LAST") {
+      dir = sql`${sql.raw(String(dir))} NULLS LAST`;
+    } else if (nullOrder === "NULLS FIRST") {
+      dir = sql`${sql.raw(String(dir))} NULLS FIRST`;
+    }
```

Comment:
The `String` cast here can probably be improved. But `sql.raw` doesn't accept an `OrderByExpression`

Reply:
Will write a test to see if we need to handle reversing here potentially too.